MoviesExample/app/src/main/java/com/novoda/demo/movies/MoviesListBinding.java

```diff
@@ -0,0 +1,16 @@
+package com.novoda.demo.movies;
+
+import android.databinding.BindingAdapter;
+import android.support.v7.widget.RecyclerView;
+
+public class MoviesListBinding {
```

Comment:
Contains a custom binding for `app:items`
Whenever the Data Binding library finds `app:items` property in a layout it will call this method.

The related xml code looks like this:
```xml
<android.support.v7.widget.RecyclerView
        android:id="@+id/movies_list"
        android:layout_width="match_parent"
        android:layout_height="match_parent"
        app:items="@{viewmodel.moviesLiveData}"
        app:layoutManager="android.support.v7.widget.LinearLayoutManager"/>
```

So what this method is going to receive is the `RecyclerView` itself and the value of `{@viewmodel.moviesLiveData}`